.github/workflows/lint.yml

```diff
@@ -17,4 +17,7 @@ jobs:
           cache: npm
       - run: yarn install --frozen-lockfile
       - run: yarn run lint
-      - run: yarn run lint-mdx
+      - name: Lint and fix MDX
```

Comment:
[optional]

```suggestion
      - name: Lint .mdx files
```

Reply:
[blocking] I don't think the CI workflow should attempt to auto fix code. Instead it should error and fail that the checked in code doesn't match pass eslint.

Reply:
agreed, thank you for reviewing this.